packages/auto-configuration-propagators/CHANGELOG.md

```diff
@@ -85,5 +86,3 @@
 ### Features
 
 * Add package for automatic propagator configuration ([#2299](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2299)) ([4bb28c9](https://github.com/open-telemetry/opentelemetry-js-contrib/commit/4bb28c99c29b52193bcd9d0f14202beac6c5dfa6))
-
-## Changelog
```

Comment:
Reviewer note: This is unrelated, but simple cleanup. There were two cases of vestigial `## Changelog` at the bottom of a file.